packages.go

```diff
@@ -71,6 +71,19 @@ func (s PackageLinks) String() string {
 	return Stringify(s)
 }
 
+// Tags holds label information about the package
+type Tags struct {
```

Comment:
The changes look fine, but lets call the type `PackageTag` instead (as its a single package tag 😏 )...

Reply:
Said and done